CSV/item-references.csv

```diff
@@ -7,4 +7,6 @@ exbl,Scalable image item reference, item ref.,HEIF
 fdel,File delivery reference,item ref.,ISO
 grid,Image item grid reference,item ref.,HEIF
 iloc,Item data location,item ref.,ISO
-thmb,Thumbnail image item reference,item ref.,HEIF
\ No newline at end of file
+prem,Pre-Mulitplied item reference,MIAF
```

Comment:
I think item ref. before the field HEIF is missing

Reply:
@denoualf , I think that is the reason why the verification script fails. Can you modify this ?